src/View.zig

```diff
@@ -0,0 +1,161 @@
+//! A View is effectively an "oversized" Window that can be written to and rendered in pieces.
+
+const std = @import("std");
+const mem = std.mem;
+
+const View = @This();
+
+const Screen = @import("Screen.zig");
+const Window = @import("Window.zig");
+const Unicode = @import("Unicode.zig");
+const Cell = @import("Cell.zig");
+
+/// View Allocator
+alloc: mem.Allocator,
+/// Underlying Screen
+screen: *Screen,
+/// Underlying Window
+win: *Window,
+
+pub const Config = struct {
+    max_width: usize = 10_000,
+    max_height: usize = 10_000,
+    x_pixel: usize = 0,
+    y_pixel: usize = 0,
+};
+pub fn init(alloc: mem.Allocator, unicode: *const Unicode, config: Config) !View {
+    const screen = try alloc.create(Screen);
+    screen.* = try Screen.init(
+        alloc,
+        .{
+            .cols = config.max_width,
+            .rows = config.max_height,
+            .x_pixel = config.x_pixel,
+            .y_pixel = config.y_pixel,
+        },
+        unicode,
+    );
+    const window = try alloc.create(Window);
```

Comment:
This `Window` shouldn't need to be allocated.

Reply:
I see what you mean. Windows are largely static bounds for rendering, but don't actually change. I'll fix that.